server/graphql/v2/mutation/MemberMutations.js

```diff
@@ -238,7 +239,7 @@ const memberMutations = {
         const host = await account.getHostCollective();
         if (host) {
           const adminCount = await models.Member.count({
-            where: { CollectiveId: account.id, role: MemberRoles.ADMIN },
+            where: { CollectiveId: { [Op.or]: [account.id, account.ParentCollectiveId] }, role: MemberRoles.ADMIN },
```

Comment:
Not sure what happens when `ParentCollectiveId` is null. Maybe we can compact this array just in case?
```suggestion
            where: { CollectiveId: { [Op.or]: compact([account.id, account.ParentCollectiveId]) }, role: MemberRoles.ADMIN },
```

Reply:
Thanks. Added. 👍🏽